genfragments/ThirteenTeV/HToZATo2L2B/Hadronizer_TuneCP5_13TeV_aMCatNLO_HToZATo2L2B_bbH-4FS_pythia8_PSweights_cff.py

```diff
@@ -0,0 +1,37 @@
+import FWCore.ParameterSet.Config as cms
+externalLHEProducer = cms.EDProducer("ExternalLHEProducer",
+    args = cms.vstring(''),
+    nEvents = cms.untracked.uint32(1000),
+    numberOfParameters = cms.uint32(1),
+    outputFile = cms.string('cmsgrid_final.lhe'),
+    scriptName = cms.FileInPath('GeneratorInterface/LHEInterface/data/run_generic_tarball_cvmfs.sh')
+)
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+from Configuration.Generator.Pythia8aMCatNLOSettings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8HadronizerFilter",
+    maxEventsToPrint = cms.untracked.int32(1),
+    pythiaPylistVerbosity = cms.untracked.int32(1),
+    filterEfficiency = cms.untracked.double(1.0),
+    pythiaHepMCVerbosity = cms.untracked.bool(False),
+    comEnergy = cms.double(13000.),
+    PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CP5SettingsBlock,
+        pythia8aMCatNLOSettingsBlock,
+        pythia8PSweightsSettingsBlock,
+        processParameters = cms.vstring(
+            'TimeShower:nPartonsInBorn = 2', #number of coloured particles (before resonance decays) in born matrix element
```

Comment:
Hi @agrohsje in this line nPartonsInBorn = 2 should be equal to 2 or 0 ?
The process is : p p > h2 > h3 Z b b~ [QCD] : b -associated production

Reply:
Also, do I need ` JetMatching:nQmatch = 4',`   4 corresponds to 4-flavour scheme (no matching of b-quarks), 5 for 5-flavour scheme ?

Reply:
this is just nlo, no additional jets. but yes if additional jets would not include b's nqmatch is 4. but this is irrelevant if there are not additional jets. npartonsinbon refers to the number of colored particles before resonance decays which i understand as 0 in your case above.

Reply:
Hi @agrohsje,
the b b~ comes from the born level if I understand this process.
<img width="322" alt="image" src="https://user-images.githubusercontent.com/17442656/122588978-aa8f6f80-d09a-11eb-9bcf-50a8a8575677.png">

Reply:
yes indeed, thanks @sihyunjeon that's exactly the diagram.

Reply:
sorry. my mistake. in this case it should be 2.